pkg/cmd/grafana-cli/commands/datamigrations/encrypt-datasource-passwords.go

```diff
@@ -0,0 +1,100 @@
+package datamigrations
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/grafana/grafana/pkg/cmd/grafana-cli/utils"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/setting"
+	"github.com/grafana/grafana/pkg/util"
+	"github.com/grafana/grafana/pkg/util/errutil"
+)
+
+func EncryptDatasourcePaswords(c utils.CommandLine, sqlStore *sqlstore.SqlStore) error {
+	var passwordRows []map[string]string
+	datasourceTypes := []string{
```

Comment:
Not sure if it is the best thing to hardcode it here, but I assume that any new datasources will be anyway using the secure way to store passwords and others like postgres already use that so no need to do anything there.

Reply:
I wonder if it makes more sense with a blacklist instead. I dont think plugins should be affected by the storage change we made in 6.2.